app/uk/gov/hmrc/brm/utils/BrmException.scala

```diff
@@ -71,6 +79,9 @@ trait BrmException extends Controller {
   }
 
   def badRequestExceptionPF(message: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
+    case e: BadRequestException if(e.message.contains("INVALID_HEADER")) =>
```

Comment:
@chris-ianson @manishsw Again, what will happen if it does not include INVALID_HEADER